### DISCUSSION TITLE
Linebreak paragraph is converted to nothing

### DISCUSSION THREAD

**Discussion by judgeNotFound** [dita-ot/dita-ot#4171] [Q&A]
In my MD documentation, people like to add line breaks at certain positions in a paragraph to avoid long lines.

e.g.,
```
One sentence.
Second sentence.
```

which is supposed to result in an output similar to this:

```
<p>
    One sentence. Second sentence.
</p>
```

Sadly, it results in the following:

```
<p>
    One sentence.Second sentence.
</p>
```
(note the missing space between both sentences)

From my observations, a single line break within an MD paragraph always converts to nothing, while it should convert to a whitespace.
How can I fix this?

**Comment by jelovirt** [comment]
See https://github.com/jelovirt/org.lwdita/issues/93

Because DITA doesn't have an element for `<br>`, LwDITA generates a `<?linebreak?>` processing-instruction that can then be converted into e.g. HTML5 `<br>` element. I have been hesitant to add support for this in DITA-OT's HTML5 and PDF transtypes, but maybe I should. Or create a separate plug-in for those who want to install it.

**Comment by jelovirt** [comment]
This will be supported by LwDITA plug-in [5.3](https://github.com/users/jelovirt/projects/9/views/1) and later on by DITA-OT 4.1.

**Reply by jelovirt** [reply]
LwDITA plug-in 5.3 has been released, install from [the plugin registry](https://www.dita-ot.org/plugins#!org.lwdita/5.3.0).

**Reply by judgeNotFound** [reply]
@jelovirt I tried doing with HTML5, where it works (is converted to `\n`). Sadly, for `--format=dita` (aka I need XML artifacts) I do not receive the desired output. Could you give further advice on that?

**Reply by jelovirt** [reply]
The DITA output is handled by [`org.dita.normalize`]( https://github.com/dita-ot/org.dita.normalize/ ) plug-in. It doesn't have an extension point to extend the XSLT to add support for `<?linebreak?>`.

To clarify, do you use _two_ trailing spaces in the end of the first line, or just line feed after the last character?

Depending which case this is, the LwDITA plug-in may need to be changed to output the line feed. Somehow… 🤔

**Reply by judgeNotFound** [reply]
We use a line feed after the last character.
